src/composables/transactionData.ts

```diff
@@ -90,16 +103,23 @@ export function useTransactionData({
    * Transaction TX function value converted into human readable label
    */
   const txFunctionLabel = computed(
-    (): string => (outerTx.value?.function)
-      ? getTxFunctionLabel(outerTx.value.function as TxFunctionRaw)
+    (): string => (txFunctionRaw.value)
+      ? getTxFunctionLabel(txFunctionRaw.value)
       : '',
   );
 
   const isTransactionCoin = computed(
     (): boolean => outerTx.value?.contractId ? isAssetCoin(outerTx.value.contractId) : true,
   );
 
-  const isDex = computed((): boolean => isTxDex(innerTx.value, dexContracts.value));
+  const isDex = computed((): boolean => (
```

Comment:
Nice finding. But I brought back the previous code because the walletConnect composable is now adding proper `tx.function` so the `isTxDex` util is recognising it properly.